app.js

```diff
@@ -282,7 +282,9 @@
       },
 
       getStringQuery: function(type) {
-        return new RegExp(this.$('.query.' + type).val(), 'i');
+        var query = this.$('.query.' + type).val();
+        var escapedQuery = query.replace(/[-[\]{}()*+?.,\\^$|#\s]/g, "\\$&");
```

Comment:
I am doing a couple of tests and it is escaping spaces too. Is this intended? https://gyazo.com/f2599c9e62870a317cb1494760abe1e0